Web_Development/FrontEnd/ownReadme.md

```diff
@@ -0,0 +1,210 @@
+                                  
+                                </div>
+                                <div class="form-group">
+                                    <div class="input-group mb-3">
+                                      <input type="text" name="Email"  value="" required="" class="form-control"placeholder="Your Email *" aria-label="Recipient's username" aria-describedby="basic-addon2">
+                                      <span class="input-group-text" id="basic-addon2">@gmail.com</span>
+                                    </div>
+                               <img src="./Googlesheet.png" alt="">
+                                </div>
+                              
+                                <div class="form-group">
+                                    <input type="number" name="Phone" class="form-control"
+                                        placeholder="Your Contact Number *" value="" required="" />
+                                </div>
+                                 <div class="form-group">
+                                    <div class="input-group mb-3">
+                                      <input type="text" name="Message"   value="" required="" class="form-control my-3" placeholder="Enter the message" aria-label="Recipient's username" aria-describedby="basic-addon2">
+                                    </div>
+                                  
+                                </div>
+                                <div class="form-group">
+                                    <button type="submit" name="submit"  class="btn btn-outline-success my-5 btn-lg">Success</button>
+                                </div>
+                                
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        const scriptURL =
+            "https://script.google.com/macros/s/AKfycbxFnHqHV20SQOIIO3jyBYcxGb2Pv2l90Xpu0u_tEN3opfMMAAdry6ShlyIEfoaS3K-jBw/exec";
+        const form = document.forms["google-sheet"];
+
+        form.addEventListener("submit", (e) => {
+            e.preventDefault();
+            fetch(scriptURL, { method: "POST", body: new FormData(form) })
+                .then((response) => alert("Thanks for fiiling the form"))
+                .catch((error) => console.error("Error!", error.message));
+        });
+    </script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+</body>
+</html>
+
+<hr>
+```
+
+ **Which basically looks like this :-**
+  <br>
+
+**Front Page UI**
+
+![Front Page UI](./FrontformUI.png)
+# Javasript code
+The below mention code which links the form with the html forms 
+<br>
+
+**Here is the javascript code**
+```
+var GooglesheetName = "Sheet1";
```

Comment:
Similarly here too. Codeblocks should have syntax

Reply:
pls check i have apply some formating to it it tooks a long time but done